src/router/routes.js

```diff
@@ -1,104 +1,122 @@
-
 const routes = [
   {
-    path: '/l',
-    component: () => import('layouts/Outside.vue'),
+    path: "/l",
+    component: () => import("layouts/Outside.vue"),
     children: [
       {
-        name: 'landing',
-        path: '',
-        component: () => import('pages/outside/Home.vue') 
+        name: "landing",
+        path: "",
+        component: () => import("pages/outside/Home.vue")
       },
       {
-        name: 'outside-notes',
-        path: 'n',
-        component: () => import('layouts/Empty.vue'),
+        name: "outside-notebook",
+        path: "nb",
+        component: () => import("layouts/Empty.vue"),
         children: [
           {
-            name: 'view-note',
-            path: ':hash',
-            component: () => import('pages/outside/ViewNote.vue'),
+            name: "view-notebooks",
+            path: ":address",
+            component: () => import("pages/outside/ViewNotebook.vue"),
             props: true
           },
+          {
+            name: "view-notebook",
+            path: ":address/:uuid",
```

Comment:
Difference in this route is that it sets a default notebook uuid selected for the notebook-list component